powa/static/js/App.vue

```diff
@@ -119,11 +119,11 @@
     </v-footer>
     <div
       class="d-flex flex-column-reverse powa-snackbars align-center"
-      style="position: fixed; bottom: 0; width: 100%"
+      style="position: fixed; bottom: 0; width: 100%; z-index: 2000"
```

Comment:
As far as I understand, this change fixes an issue with the snackbar being hidden behind the footer. I'd prefer we explicitely set the `z-index` to `initial` in the `<v-footer>` instead.